packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -241,6 +245,9 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
             async (fullGC?: boolean) => this.getGCDataInternal(fullGC),
             async () => this.getInitialGCSummaryDetails(),
         );
+
+        this.logger = ChildLogger.create(_containerRuntime.logger, "FluidDataStoreContext");
```

Comment:
I think this would change how all the logs would show up for data stores / dds, as this logger would be used by data store to create its child loggers.
Instead you likely want to keep this.logger as is, but create private child logger for internal needs

Reply:
Good point